pkg/proxy/integrations/postgresParser/postgres_parser.go

```diff
@@ -453,6 +465,12 @@ func decodePostgresOutgoing(requestBuffer []byte, clientConn, destConn net.Conn,
 			continue
 		}
 
+		tcsMocks := h.GetTcsMocks()
+		// change auth to md5 instead of scram
+		// CheckValidEncode(tcsMocks, h, logger)
+		ChangeAuthToMD5(tcsMocks, h, logger)
```

Comment:
Please consult with @Sarthak160 regarding this change I think keeping 
ChangeAuthToMD5 here may cause some problem.

Reply:
I think this should'nt cause any problem since we are changing the mocks before writing back to client . Hey @404fixer, if this is running in test-mode via any sample app , you can keep these changes .